src/gxcore/social/share/shareText.ts

```diff
@@ -0,0 +1,24 @@
+import { msg } from "../../../misc/msg";
+
+/**
+ * @param text
+ * @param url
+ * @param title
+ */
+export const shareText = (text: string, url: string, title: string) => {
+  const nav: any = window.navigator;
+  if (nav && nav.share) {
+    nav
+      .share({
+        title: title,
```

Comment:
Nitpicking...

```suggestion
        title,
```

Reply:
Would that really work? As I understand it, this code is constructing an object. The first `title:` is the property name, the second `title` is the variable containing the value to store in that property. If I remove the property name, it will not work... Right?

Reply:
It's a shorthand for setting properties (see more [here](http://es6-features.org/#PropertyShorthand)). TypeScript supports it.

Reply:
Thanks, didn't know that... Fixed @ [162d441](https://github.com/genexuslabs/web-standard-functions/pull/61/commits/162d44104d3ca8349831864d45f2e65a9ee0ee6f)